be/src/exec/CMakeLists.txt

```diff
@@ -213,19 +213,7 @@ set(EXEC_FILES
     pipeline/query_context.cpp
     pipeline/stream_epoch_manager.cpp
     pipeline/stream_pipeline_driver.cpp
-    pipeline/aggregate/aggregate_blocking_sink_operator.cpp
-    pipeline/aggregate/aggregate_blocking_source_operator.cpp
-    pipeline/aggregate/aggregate_streaming_sink_operator.cpp
-    pipeline/aggregate/aggregate_streaming_source_operator.cpp
-    pipeline/aggregate/aggregate_distinct_blocking_sink_operator.cpp
-    pipeline/aggregate/aggregate_distinct_blocking_source_operator.cpp
-    pipeline/aggregate/aggregate_distinct_streaming_sink_operator.cpp
-    pipeline/aggregate/aggregate_distinct_streaming_source_operator.cpp
-    pipeline/aggregate/sorted_aggregate_streaming_sink_operator.cpp
-    pipeline/aggregate/sorted_aggregate_streaming_source_operator.cpp
-    pipeline/aggregate/spillable_aggregate_blocking_sink_operator.cpp
-    pipeline/aggregate/spillable_aggregate_blocking_source_operator.cpp
-    pipeline/aggregate/spillable_aggregate_distinct_blocking_operator.cpp
+    pipeline/aggregate/combined.cpp
```

Comment:
combined.cpp is confusing,  suggest to use operators.cpp or agg_operators.cpp

Reply:
done